src/TSTransformer/nodes/class/transformDecorators.ts

```diff
@@ -141,22 +141,28 @@ function transformParameterDecorators(
 ): luau.List<luau.Statement> {
 	const result = luau.list.make<luau.Statement>();
 
+	const memberName = member.name;
+	if (!memberName || ts.isPrivateIdentifier(memberName)) return result;
```

Comment:
Because of having no name, constructors should call the decorator with `undefined` (nil) as the second parameter. TypeScript verifies the types for this when a parameter decorator is used in the constructor.